get-started/what-is-shiny.qmd

```diff
@@ -0,0 +1,167 @@
+  .btn-links {
+    display: flex;
+    gap: 1.5rem;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .btn-link-large {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 150px;
+    height: 150px;
+    font-size: 1.5rem;
+    border: 1px solid transparent;
+    border-radius: 10px;
+    cursor: pointer;
+    padding: 1.25rem;
+    text-decoration: none;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  }
+
+  .btn-link-large i {
+    margin-top: 10px;
+    font-size: 2rem;
+  }
+
+  .btn-link-large:hover {
+    text-decoration: none;
+    border: 1px solid var(--bs-primary);
+  }
+</style>
+```
+
+Whether you’re a data scientist, analyst, or developer, Shiny makes it easy to [create](create-run.qmd) rich, [interactive experiences](/docs/ui-overview.qmd) in pure Python with a fully [reactive framework](/docs/reactive-foundations.qmd).
+No need to learn JavaScript or front-end frameworks.
+
+## Batteries included {#batteries-included}
+
+Shiny for Python comes fully equipped with everything you need to build a dashboard right out of the box, including a rich set of [input](/components/#inputs) and [output](/components/#outputs) components.
+There is an entire [components gallery](/components/) to help you quickly build interactive applications.
+
+[Layout](/layouts/) options let you organize your UI efficiently, while built-in [theming](/docs/ui-customize.qmd) (including [dark mode](/components/inputs/dark-mode)) ensures your app looks great with minimal effort.
+
+Already have a brand guideline?
+You can use [brand.yml](https://posit-dev.github.io/brand-yml/) to apply consistent branding, colors, and logos across your application.
+
+<div class="btn-links">
+  <a href="/components#inputs" class="btn-link-large">
+    Inputs
+    <i class="bi bi-sliders primary" role="img" aria-hidden="true"></i>
+  </a>
+
+  <a href="/components#outputs" class="btn-link-large">
+    Outputs
+    <i class="bi bi-bar-chart-line-fill primary" role="img" aria-hidden="true"></i>
+  </a>
+
+  <a  href="/layouts" class="btn-link-large">
+    Layouts
+    <i class="bi bi-layout-text-window-reverse primary" role="img" aria-hidden="true"></i>
+  </a>
+</div>
+
+## Reactivity {#reactivity}
+
+At the heart of Shiny is [reactivity](/docs/reactive-foundations.qmd), a system that automatically updates [outputs](/components/#outputs) when [inputs](/components/#inputs) change for seamless interactivity, without callbacks.
+
+Shiny's reactive engine avoids unnecessary computations by only re-calculating the outputs whose inputs have changed, making Shiny ideal for fast data-driven applications, enabling live updates for [charts, tables](/components/#outputs), and [reports](https://quarto.org/docs/interactive/shiny/) with minimal effort.
+
+Below is a Shiny application where you have the output text react to _both_ input slider values
+and the second row of text _only_ reacts to the first slider value.
```

Comment:
maybe add another sentence: Try updating the sliders below. It's a live code cell, you can even update the app if you want.